### ISSUE TITLE
Adds clean/purge data button in the troubleshooting page

### ISSUE THREAD

**Issue by benoitf** [containers/podman-desktop#5022]
### Is your enhancement related to a problem? Please describe

Add cleanup operation for each provider supporting the handling.

### Describe the solution you'd like

I would say that we can select which provider we might want to cleanup (only the one supporting may be listed ?)

### Additional context

part of epic https://github.com/containers/podman-desktop/issues/4994

**Comment by benoitf** [comment]
![image](https://github.com/containers/podman-desktop/assets/436777/3c0a0596-6b3c-487a-903d-acc1d43050d0)

cc @ekidneyrh @mairin

**Comment by mairin** [comment]
Notes from the UX meeting today:

- Let's switch the Troubleshooting page to be tab-based.
- First tab will be Repair & Connections
- Second tab will be Logs
- Third tab will be Stores
- We also talked about dropping the broom icon from the subtitle and leaving it bare. This removes the duplication of the broom icon between the subtitle and button (and maybe gives a cleaner look.)
- We talked about adding a little bit more padding between the "Clean / Purge data" subtitle and the warning text below. And a little bit more padding at the bottom of the repair card.

![Screenshot from 2023-12-07 10-01-12](https://github.com/containers/podman-desktop/assets/799683/163cabff-ad23-4c1c-9651-de0aefdb2680)
